.github/renovate.json5

```
@@ -229,6 +232,13 @@
         "quay.io/cilium/hubble"
       ],
     },
+    {
+      "groupName": "Go",
+      "matchDepNames": [
+        "go",
+        "docker.io/library/golang"
+      ]
+    },
```

could we do:
```suggestion
    {
      "groupName": "Go",
      "matchDepNames": [
        "go",
        "docker.io/library/golang"
      ],
      "schedule": [
        "on friday"
      ]
    },
```
image SHAs for `docker.io/library/golang` keeps changing even for the same patch version (see https://github.com/cilium/cilium/pull/24846), and renovate keeps opening new pull requests 🥺

---

Good point, done.